sonar-project.properties

```diff
@@ -0,0 +1,16 @@
+sonar.projectKey=smartcontractkit_chainlink-data-streams_AYwW9oTeBZUeim9slK4s
```

Comment:
Do we need to generate a new project key with the name change from `llo` to `data-streams`?

Reply:
yeah it seems like we need to setup a whole new project when there is repo rename. I just did it and updated it.